requirements.txt

```
@@ -1,3 +1,4 @@
+dataclasses;python_version<"3.7"
```

```suggestion
dataclasses; python_version<"3.7"
```
add space after `;` for readibility

---

@bzantium 반영했습니다!